dependencies/buster/puppet-strings/changelog

```diff
@@ -1,3 +1,9 @@
+ruby-puppet-strings (1.1.0-2) stable; urgency=low
```

Comment:
I don't see any change here (same for focal)

Reply:
that's correct, it was already using the "real" package there, but I wanted the versions to align.

Reply:
my plan is to get rid of most contents in the focal and bionic folders, unless the builds *really* differ from buster, but for that they need to match first ;)

Reply:
Fair enough.